templates/rollup-template/rollup_config.toml

```diff
@@ -0,0 +1,23 @@
+[da]
+# The JWT used to authenticate with the celestia light client. Instructions for generating this token can be found in the README
+celestia_rpc_auth_token = "MY.SECRET.TOKEN"
```

Comment:
What are your thoughts on using `MockDa`. This way our unit test also test that if the `main` works.

Reply:
I think it's probably preferable to use a local Celestia devnet like we do for demo-rollup. I was planning that for a follow-up - @cemozerr would you prefer a Mock instead? It's easy to switch, but I feel like it's nice for users to see the real thing instead of a mock

Reply:
I actually think it'd be better to have mock DA for demo-rollup too. Easier to spin up the better IMO.

Reply:
Ok, switched to mocks